Controllers/AssetController.cs

```diff
@@ -26,5 +26,57 @@ public IEnumerable<Asset> Get()
             return assets;
         }
     }
-}
 
+
+    [HttpPost(Name = "CreateAsset")]
+    public async Task<IActionResult> Create([FromBody] Asset asset)
+    {
+        using (var connection = _dapperContext.GetConnection())
+        {
+            var sqlStatement = @"
+            INSERT INTO Assets 
+                (TypeId,
+                Location,
+                EnglishName,
+                CopticName)
+            VALUES (@TypeId,
+                @Location,
+                @EnglishName,
+                @CopticName)";
+            await connection.ExecuteAsync(sqlStatement, asset);
```

Comment:
is there a reason these operations are `async`?
I'm all for that if we are executing other code meanwhile, this would be perfect, but in this case, it's the only thing happening.
Is there a reason you chose to do that?

Reply:
It probably isnt required for these APIs

Reply:
I dont have a full grasp of how it works in C# but i believe when the function is async the caller doesnt get blocked

Reply:
Does it make sense to create a dao or data access layer behind these APIs?

Reply:
Like we discussed in the car, we might not need one, but if we need one, we'll create one then.